.github/stale.yml

```
@@ -1,41 +1,9 @@
-
-    If your PR wasn't reviewed at all, it might help to find someone who's perhaps a user of the package or module you are changing, or alternatively, ask once more for a review by the maintainer of the package/module this is about. If you don't know any, you can use [Git blame](https://git-scm.com/docs/git-blame) on the relevant files, or [GitHub's web interface](https://docs.github.com/en/github/managing-files-in-a-repository/tracking-changes-in-a-file) to find someone who touched the relevant files in the past.
-
-    If your PR has had reviews and nevertheless got stale, make sure you've responded to all of the reviewer's requests / questions. Usually when PR authors show responsibility and dedication, reviewers (privileged or not) show dedication as well. If you've pushed a change, it's possible the reviewer wasn't notified about your push via email, so you can always [officially request them for a review](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/requesting-a-pull-request-review), or just @ mention them and say you've addressed their comments.
-
-    Lastly, you can always ask for help at [our Discourse Forum](https://discourse.nixos.org/), or more specifically, [at this thread](https://discourse.nixos.org/t/prs-in-distress/3604) or at [#nixos' IRC channel](https://webchat.freenode.net/#nixos).
-
-issues:
-  markComment: |
-    Hello, I'm a bot and I thank you in the name of the community for opening this issue.
-
-    To help our human contributors focus on the most-relevant reports, I check up on old issues to see if they're still relevant. This issue has had no activity for 180 days, and so I marked it as stale, but you can rest assured it will never be closed by a non-human.
-
-    The community would appreciate your effort in checking if the issue is still valid. If it isn't, please close it.
-
-    If the issue persists, and you'd like to remove the stale label, you simply need to leave a comment. Your comment can be as simple as "still important to me". If you'd like it to get more attention, you can ask for help by searching for maintainers and people that previously touched related code and @ mention them in a comment. You can use [Git blame](https://git-scm.com/docs/git-blame) or [GitHub's web interface](https://docs.github.com/en/github/managing-files-in-a-repository/tracking-changes-in-a-file) on the relevant files to find them.
-
-    Lastly, you can always ask for help at [our Discourse Forum](https://discourse.nixos.org/) or at [#nixos' IRC channel](https://webchat.freenode.net/#nixos).
-
-# Comment to post when closing a stale issue. Set to `false` to disable
+markComment: inactive 180 days → stale; I will not close. New to stale bot? See [Nixpkgs stale bot info](https://github.com/NixOS/nixpkgs/blob/master/.github/STALE-BOT.md).
```

```suggestion
markComment: When an issue or PR is inactive for 180 days, I label it stale. I will not close it, and I'll remove the label if anyone replies. Need help? Read my [recommendations for stale Nixpkgs issues & PRs](https://github.com/NixOS/nixpkgs/blob/master/.github/STALE-BOT.md).
```
Hopefully this is a bit of a Goldilocks zone? I'm trying to hit a few things:
- friendly and helpful
- to-the-point
- not requiring a click for the PR/issue fountain-of-youth (it seems possible you're very-intentionally trying to make them click for this, though--so it may be worth haggling over! :)

---

Yeah, I'm not so sure about this one. I'm trying to make it super short for a bunch of reasons:

1. We can't change all the old comments it makes if something changes
2. We'd like people to read the more detailed page
3. We don't want to annoy regular contributors with a long message

---

I'm almost wondering if we shouldn't put anything in the comment but a link to the Markdown doc. Like:

```suggestion
markComment: [Nixpkgs stale bot info](https://github.com/NixOS/nixpkgs/blob/master/.github/STALE-BOT.md).
```

---

Yes please!

- The bot name has it all.
- The github history already makes it's actions clear.

> Conciseness is the real respectfulness.

---

I guess I've been primed by TL;DR, which doesn't quite square with giving almost no context, and a link to a document they still have to read (but I agree that the document itself is more direct, now).

I'm not opposed to it just being a link, though I am starting to think issues and PRs should link to separate sections of the document? It may also make sense for "issues" to have separate links (like, "problem reports" and "suggestions/requests"?), but I'll address this a little more directly on the other section.

---

Updated it to be just the link. Yeah, I suppose separate links for issues and PRs might be nice because the individual documents could be shorter, but that seems too fancy for now.